src/entities/Sto.ts

```diff
@@ -262,10 +320,10 @@ export abstract class Sto<P> extends Entity<P> {
   };
 
   /**
-   * Removes a role from a delegate
+   * Remove a role from a delegate
    *
-   * @param delegateAddress wallet address of the delegate
-   * @param role role to revoke
+   * @param args.delegateAddress - wallet address of the delegate
+   * @param args.role - role to revoke
```

Comment:
"Arg properties which are complex interfaces defined elsewhere should not be broken down into their own properties (the autogen tool provides a link to the corresponding interface)"

Reply:
Not sure I follow, where elsewhere are these defined?

Reply:
StoRole is an enum type, my bad.